hyperspy/models/edsmodel.py

```diff
@@ -318,6 +318,62 @@ def disable_xray_lines(self):
         for component in self._active_xray_lines:
             component.active = False
 
+    def fit(self, fitter='mpfit', method='ls', grad=True,
+            bounded=True, ext_bounding=False, update_plot=False,
+            **kwargs):
+        """Fits the EDS model to the experimental data.
+
+        The fitting of an EDS model is, by default, performed by bounded
+        optimisation using mpfit. This constrains the components to be positive
+        as is physical. Alternative fitters may still be specified.
+
+        Parameters
+        ----------
+        fitter : {None, "leastsq", "odr", "mpfit", "fmin"}
+            The optimizer to perform the fitting. By default mpfit is used for
+            EDS models, as this supports bounding to force the components in the
+            model to be positive. This is inline with the physical nature of EDS
+            spectra. All optimizers available for Model1D remain availble and
+            can be specified by the user. fmin is the only one that supports
+            maximum likelihood estimation, but it is less robust than leastsq
```

Comment:
I take issue with the original, but I'll raise that separately. Personally I would leave out any mention of "robustness" at all since a) it's unclear what that means here and b) the take-home point is surely MLE for Poisson noise.